plane/source/docs/quadplane-parameters.rst

```diff
@@ -29,6 +29,36 @@ Key Parameters
    throttle while flying or you risk disarming your motors.
 -  The default :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>` for a QuadPlane is to 300 (Hz). Most QuadPlanes do not need this to be rasied. Some very small vehicles (< 1Kg) might benefit from setting it to 400. In heavier vehicles, their higher inertia results in lower effective control response rates, so they do not benefit from a higher loop rate. Raising above 300 only leads to larger log files in these vehicles.
 
+.. _return_behavior_setup:
+
+Return Behavior QuickSetup Guide:
+=================================
+
+While there are many parameters setting distances and heights for the various home return modes/behaviors (read each mode's wiki page), this is a quick setup guide for basic behaviors.
+
+1.If you lose RC link for greater than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` in fixed wing flight set :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` value below to obtain various behavior options:
```

Comment:
I think the word "below" doesn't make sense since it's actually right here. For "below" I'd expect to have to scroll to find the "below".